returnn/hdf.py

```diff
@@ -309,12 +309,27 @@ def __init__(
         self.allophone_file = allophone_file
         self.state_tying_file = state_tying_file
         self.out_hdf_files = [self.output_path(f"data.hdf.{d}", cached=False) for d in range(len(alignment_caches))]
+        self.excluded_segment_sublists = [
```

Comment:
I understand that we need to write this in files before merge due to what @michelwi said, but do we want these files to be outputs? I would rather suggest writing them into the work folder.